tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -136,8 +138,13 @@ class MklDnnConvUtil {
       CHECK_BOUNDS(input_cols_raw, "Input cols too large");
       int input_cols = static_cast<int>(input_cols_raw);
 
+#ifdef ENABLE_MKLDNN_V1
+      // MKL-DNN always requires input in NCHW format Conv2D.
+      std::vector<long int> mkldnn_sizes(4, -1);
```

Comment:
You can define `MKLDNN_SIZE_DTYPE` (or any name you prefer) to be either `long int` or `int` at the beginning and then use it throughout the file.

```suggestion
      std::vector<MKLDNN_SIZE_DTYPE> mkldnn_sizes(4, -1);
```

Reply:
Done